config/sets/annotations-to-attributes.php

```diff
@@ -88,7 +88,6 @@
         new AnnotationToAttribute('afterClass', 'PHPUnit\Framework\Attributes\AfterClass'),
         new AnnotationToAttribute('before', 'PHPUnit\Framework\Attributes\Before'),
         new AnnotationToAttribute('beforeClass', 'PHPUnit\Framework\Attributes\BeforeClass'),
-        new AnnotationToAttribute('codeCoverageIgnore', 'PHPUnit\Framework\Attributes\CodeCoverageIgnore'),
```

Comment:
Could you add links to the 2 issues and not about code coverage being deprecated? 
Just so someone don't re-add it here in near future.

Reply:
Add links where? In code?

Reply:
@martinssipenko probably comment the line instead of remove and add `@see` link doc

Reply:
Here in this line